src/medCore/gui/medAbstractWorkspace.h

```diff
@@ -75,6 +75,7 @@ class MEDCORE_EXPORT medAbstractWorkspace : public QObject
     bool isUserLayerPoolable() const;
     bool isUserViewPoolable() const;
     bool isUserLayerClosable() const;
+    virtual void setInitialGroups();
```

Comment:
This function is virtual and is being called in the constructors. This is generally a bad idea (see https://isocpp.org/wiki/faq/strange-inheritance#calling-virtuals-from-ctors). If you have virtual initialisation functions, one solution is to call them after creation of the instance. This is how ```setupViewContainerStack``` (another a virtual initialisation function) is being handled. It is being called in ```medWorkspaceArea::setupWorkspace``` after instanciation of the workspace.